src/specklepy/objects/geometry/control_point.py

```diff
@@ -0,0 +1,26 @@
+from dataclasses import dataclass
+
+from specklepy.objects.base import Base
+from specklepy.objects.interfaces import IHasUnits
+
+
+@dataclass(kw_only=True)
+class ControlPoint(Base, IHasUnits, speckle_type="Objects.Geometry.ControlPoint"):
```

Comment:
In C#, this class inherits `Point`, is there a reason we wouldn't want that here in py?

Reply:
didn't want to create an inheritance chain between object itself, if you see any significant value, we can update it.